alacritty_terminal/src/term/mod.rs

```diff
@@ -1315,10 +1315,9 @@ impl<T: EventListener> ansi::Handler for Term<T> {
     }
 
     #[inline]
-    fn dectest(&mut self) {
-        trace!("Dectesting");
-        let mut template = self.cursor.template;
-        template.c = 'E';
+    fn decaln(&mut self) {
+        trace!("Decalnning");
+        let template = Cell { c: 'E', ..self.cursor.template };
```

Comment:
Trying out both XTerm and Termite, none of them seem to set the background color based on the template cell. We might just want to use `..Cell::default()` here?

Reply:
Oh, I've forgotten to check background thing in some escapes. Let me recheck all the thing.

Reply:
Seems like `decaln` was the only one affected.